src/main/java/org/opensearch/security/compliance/ComplianceConfig.java

```diff
@@ -235,6 +235,7 @@ public static ComplianceConfig from(Map<String, Object> properties, @JacksonInje
         final Set<String> ignoredComplianceUsersForRead = ImmutableSet.copyOf(
             getOrDefault(properties, "read_ignore_users", AuditConfig.DEFAULT_IGNORED_USERS)
         );
+
```

Comment:
nit: extra line. (in couple other files as well)

Reply:
Thanks DC, missed the test renames. Added the newlines and fixed the old comment I bumped.

Reply:
Missing a push? I still see these